open-company/open-company-web
# Support

Change how What's New and Support links in the menu are handled.
Before What's New was loading the background the carrot/what-s-new section content via our API and we used the `whats-new-modal` component to display it.
Now we linked it to https://beta.carrot.io/carrot-support/what-s-new.
Also we linked Help in the menu with https://beta.carrot.io/carrot-support/all-posts.

Related PR: https://github.com/open-company/open-company-storage/pull/126

To test:
- start storage branch
- start this web branch
- login or signup
- once you are logged in go to AP
- check in the Network tab the storage entry point response
- [ ] does it NOT contain a url with ref `whats-new `? Good
- [ ] do you NOT see an attempt to load a what's new section data via our API? Good
- open the menu
- click on Support
- [ ] do you see https://beta.carrot.io/carrot-support/all-posts opening in a new tab/window? Good
- click on What's New
- [ ] do you see https://beta.carrot.io/carrot-support/what-s-new opening in a new tab/window? Good
